langchain-core/src/runnables/passthrough.ts

```diff
@@ -1,206 +1,12 @@
-    options?: Partial<RunnableConfig>
-  ): AsyncGenerator<RunOutput> {
-    return this._transformStreamWithConfig(
-      generator,
-      this._transform.bind(this),
-      options
-    );
-  }
-
-  async stream(
-    input: RunInput,
-    options?: Partial<RunnableConfig>
-  ): Promise<IterableReadableStream<RunOutput>> {
-    async function* generator() {
-      yield input;
-    }
-    return IterableReadableStream.fromAsyncGenerator(
-      this.transform(generator(), options)
-    );
-  }
-}
-
-export interface RunnablePickFields {
-  keys: string | string[];
-}
-
-/**
- * A runnable that assigns key-value pairs to inputs of type `Record<string, unknown>`.
- */
-export class RunnablePick<
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    RunInput extends Record<string, any> = Record<string, any>,
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    RunOutput extends Record<string, any> | any = Record<string, any> | any,
-    CallOptions extends RunnableConfig = RunnableConfig
-  >
-  extends Runnable<RunInput, RunOutput>
-  implements RunnablePickFields
-{
-  static lc_name() {
-    return "RunnablePick";
-  }
-
-  lc_namespace = ["langchain_core", "runnables"];
-
-  lc_serializable = true;
-
-  keys: string | string[];
-
-  constructor(fields: string | string[] | RunnablePickFields) {
-    if (typeof fields === "string" || Array.isArray(fields)) {
-      // eslint-disable-next-line no-param-reassign
-      fields = { keys: fields };
-    }
-    super(fields);
-    this.keys = fields.keys;
-  }
-
-  async _pick(input: RunInput): Promise<RunOutput> {
-    if (typeof this.keys === "string") {
-      return input[this.keys];
-    } else {
-      const picked = this.keys
-        .map((key) => [key, input[key]])
-        .filter((v) => v[1] !== undefined);
-      return picked.length === 0 ? undefined : Object.fromEntries(picked);
-    }
-  }
-
-  async invoke(
-    input: RunInput,
-    options?: Partial<CallOptions>
-  ): Promise<RunOutput> {
-    return this._callWithConfig(this._pick.bind(this), input, options);
-  }
-
-  async *_transform(
-    generator: AsyncGenerator<RunInput>
-  ): AsyncGenerator<RunOutput> {
-    for await (const chunk of generator) {
-      const picked = await this._pick(chunk);
-      if (picked !== undefined) {
-        yield picked;
-      }
-    }
-  }
-
-  transform(
-    generator: AsyncGenerator<RunInput>,
-    options?: Partial<RunnableConfig>
-  ): AsyncGenerator<RunOutput> {
-    return this._transformStreamWithConfig(
-      generator,
-      this._transform.bind(this),
-      options
-    );
-  }
-
-  async stream(
-    input: RunInput,
-    options?: Partial<RunnableConfig>
-  ): Promise<IterableReadableStream<RunOutput>> {
-    async function* generator() {
-      yield input;
-    }
-    return IterableReadableStream.fromAsyncGenerator(
-      this.transform(generator(), options)
-    );
-  }
-}
+export { RunnableAssign };
```

Comment:
I think you can remove this export? Only entrypoint is through `index.ts`

Reply:
Done